float32 ax
float32 ay
float32 az
float32 gx
float32 gy
float32 gz
float32 sysCal
float32 accelCal
float32 gyroCal
float32 magCal
float32 dt
